internal/redis/dump/dump.go

```diff
@@ -26,12 +27,15 @@ func (f *dump) Apply(ctx *sqlite.Context, values ...sqlite.Value) {
 
 	result, err := f.rdb.Dump(context.TODO(), key).Result()
 	if err != nil {
+		if errors.Is(err, redis.Nil) {
+			ctx.ResultNull()
+			return
+		}
```

Comment:
`If key does not exist a nil bulk reply is returned.` (rather than bubble the error up all the way, handle the special case when it's a `NIL` response from redis and just return `NULL`)